core/src/main/java/org/openstack4j/openstack/networking/domain/NeutronNetwork.java

```diff
@@ -52,9 +51,11 @@
     @JsonProperty("availability_zones")
     private List<String> availabilityZones;
     @JsonProperty("created_at")
-    private String createdTime;
+    @JsonFormat(pattern="yyyy-MM-dd'T'HH:mm:ss")
```

Comment:
Is specifying the format really needed? A lot of date fields is not having those at all.

Reply:
This is not really a format that must be specified.

Reply:
Thank you for your suggestion. I haven't considered it comprehensively.